Create README.md (#1)
magic-with-latents/.github: profile/README.md

## Hi there 👋

> The organization aims to explain modern and complex ML tech to the community. Understanding any ML component comprehensively involves of three things:

Reviewer: Org is a powerful word. I will avoid using it as much as possible. Maybe something like this?

We aim to explain new and complex ML tech to the community. This is our way of giving back to the awesome ML community. Understanding....

> The organization aims to explain modern and complex ML tech to the community. Understanding any ML component comprehensively involves of three things:

Author: ```suggestion
We aim to explain new and complex ML tech to the community. This is our way of giving back to the community. Understanding any ML component comprehensively involves three things:
```

Reviewer: LGTM!

* Underlying math
* Conceptual framework of the component
* Corresponding code 

> We're starting with a series on Diffusion models where there will be deep dives into the fundamental math behind them as well as self-contained and readable code. 

Reviewer: where we will deep dive*

> We're starting with a series on Diffusion models where there will be deep dives into the fundamental math behind them as well as self-contained and readable code. 

Author: ```suggestion
We're starting with a series on Diffusion models where we will dive deep into the fundamental math behind them as well as provide self-contained and readable code. 
```

Reviewer: LGTM!
